zombies.js

```diff
@@ -78,6 +83,23 @@
  * @property {method} getPack              Returns private variable `pack`.
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
+function Player( name, health, strength, speed) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  var pack = [];
```

Comment:
Private variables have a specific naming convention, it is not being used here. Also, the way it is currently setup, the private variable is not accessable outside of the constructor.